pants.ini

```diff
@@ -402,7 +402,7 @@ resolver_cache_dir: %(pants_bootstrapdir)s/python_cache/requirements
 version: pytest>=5.2.4
 # TODO(#8651): We need this until we switch to implicit namespace packages so that pytest-cov
```

Comment:
This comment probably now refers only to the second item in the list, so should probably break the list onto multiple lines to put it closer to that item.

Additionally, it would be good to refer to https://github.com/pantsbuild/pants/issues/8528 via a TODO here.